pycode style not  found

when `ioda-converters` is part of a bundle, the included `pycodestyle` file is not found. Sometimes a system `pycodestyle` is used instead, which explains why in the past I had seen differences in the coding norms.

PYCODESTYLE_CFG_PATH is pointing to the root of my soca-bundle source dir

```
1: Test command: /home/tsluka/work/jedi-soca/soca.build/bin/lint.sh "/home/tsluka/work/jedi-soca/soca.src/ioda-converters/src/lib-python" "/home/tsluka/work/jedi-soca/soca.src"
1: Environment variables:
1:  OMP_NUM_THREADS=1
1: Test timeout computed to be: 9.99988e+06
1: + SRC_DIR=/home/tsluka/work/jedi-soca/soca.src/ioda-converters/src/lib-python
1: + PYCODESTYLE_CFG_PATH=/home/tsluka/work/jedi-soca/soca.src
1: + cd /home/tsluka/work/jedi-soca/soca.src/ioda-converters/src/lib-python
1: + pycodestyle --config=/home/tsluka/work/jedi-soca/soca.src/.pycodestyle .
1: ./ioda_conv_ncio.py:11:80: E501 line too long (83 > 79 characters)
1: ./ioda_conv_ncio.py:13:80: E501 line too long (83 > 79 characters)
1: ./ioda_conv_ncio.py:44:80: E501 line too long (83 > 79 characters)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,6 +70,9 @@ if( HAVE_ODB_API )
   list( APPEND IODACONV_EXTRA_LIBRARIES     Odb    )
 endif()
 
+# Location of .pycodestyle for norm checking within IODA-converters
+set( IODACONV_PYLINT_CFG_DIR ${CMAKE_CURRENT_SOURCE_DIR} )
+
 ################################################################################
 # Sources
 ################################################################################

--- a/src/ODB2/CMakeLists.txt
+++ b/src/ODB2/CMakeLists.txt
@@ -19,4 +19,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_odb2_coding_norms
                   TYPE SCRIPT
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

--- a/src/gsi-ncdiag/CMakeLists.txt
+++ b/src/gsi-ncdiag/CMakeLists.txt
@@ -22,4 +22,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_gsi-ncdiag_coding_norms
                   TYPE SCRIPT
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

--- a/src/lib-python/CMakeLists.txt
+++ b/src/lib-python/CMakeLists.txt
@@ -21,4 +21,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_lib-python_coding_norms
                   TYPE SCRIPT
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

--- a/src/marine/CMakeLists.txt
+++ b/src/marine/CMakeLists.txt
@@ -26,4 +26,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_marine_coding_norms
                   TYPE
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

--- a/src/ncep/CMakeLists.txt
+++ b/src/ncep/CMakeLists.txt
@@ -18,4 +18,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_ncep_coding_norms
                   TYPE SCRIPT
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

--- a/src/utilities/CMakeLists.txt
+++ b/src/utilities/CMakeLists.txt
@@ -16,4 +16,4 @@ install (PROGRAMS
 ecbuild_add_test( TARGET iodaconv_utils_coding_norms
                   TYPE
                   COMMAND ${CMAKE_BINARY_DIR}/bin/lint.sh
-                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR} )
+                  ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${IODACONV_PYLINT_CFG_DIR} )

